### DISCUSSION TITLE
booting mid-2007 MacBook with 32-bit EFI into Ventoy

### DISCUSSION THREAD

**Discussion by chinarut** [ventoy/Ventoy#2059] [Q&A]
hi! I verified my Ventoy flash drive on an 2008 iMac and I love the persistent files support with .img files!

I got word that Ventoy might help me boot ISO images on my mid-2007 MacBook which has a 32-bit EFI.

Unfortunately the MacBook doesn’t recognize my Ventoy flash drive [on boot] like it did on my iMac - it just goes right into OS X.

Does anyone have this working here by chance?

If I can find the website of a guy who wrote a utility to modify ISOs so when you burn them, they successfully boot Live CDs into a mid-2007 MacBook, I’ll circle back around and provide more info in case it helps evolve Ventoy a bit more :)

**Comment by steve6375** [comment]
What type of Ventoy disk did you make?
MBR or GPT?
Secure Boot or Non-Secure Boot?
Did you try all four combinations?

**Reply by chinarut** [reply]
hey sorry for dropping the ball here.

I just tried to boot into the stick I've been using on all my Macs and it works.

1.0.84 and it says "IA32" so it's working!

I'll close this issue.
